src/ui/public/agg_types/buckets/date_histogram.js

```diff
@@ -41,7 +41,8 @@ export default function DateHistogramAggType(timefilter, config, Private) {
     makeLabel: function (agg) {
       const output = this.params.write(agg);
       const params = output.params;
-      return params.field + ' per ' + (output.metricScaleText || output.bucketInterval.description);
+      const field = params.field || agg.params.field.displayName || '';
```

Comment:
I couldn't trigger a single example where the `agg.params.field.displayName` was empty. Shouldn't we just use that instead of using the cascade?